### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by tfrank64**
- Using CloudConfiguration now
- Added public web view
- Updated packages and Readme

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<!--
+ Copyright IBM Corporation 2015
+
```
Comment (jkingoliver) [inline public/index.html]: Update copyright to 2017

File: public/welcome.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
```
Comment (jkingoliver) [inline public/welcome.css]: update copyright

File: Sources/Bookstore/Config.swift [hunk 3]
```diff
@@ -14,59 +14,40 @@
...
 
     static let sharedInstance = Config()
     
     var port: Int
-    var ip: String
 	var databaseHost = "localhost"
 	var databasePort = Int32(5432)
-	var userName     = "rfdickerson"
+	var userName     = "tlfrankl"
 	var password     = "password"
```
Comment (jkingoliver) [inline Sources/Bookstore/Config.swift]: Do you want your/Robert's userName hardcoded here or would it be better to just add a placeholder?

Comment (tfrank64) [inline Sources/Bookstore/Config.swift]: Probably a placeholder, I will update this PR tomorrow

Comment (rfdickerson) [inline Sources/Bookstore/Config.swift]: You might want to use the Configuration library, instead, now that we have it.

Comment (tfrank64) [inline Sources/Bookstore/Config.swift]: We actually are using the Configuration library, these are just defaults if not available or doing local dev.

**Pull request merged**